cardano-testnet/cardano-testnet.cabal

```diff
@@ -57,27 +57,28 @@ library
                       , unordered-containers
 
   hs-source-dirs:       src
-  exposed-modules:      Parsers
-                        Parsers.Babbage
-                        Parsers.Byron
-                        Parsers.Cardano
-                        Parsers.Shelley
-                        Parsers.Version
+  exposed-modules:      Cardano.Testnet
+                        Testnet.Byron
                         Util.Assert
                         Util.Base
                         Util.Ignore
                         Util.Process
                         Util.Runtime
+                        Parsers
```

Comment:
I don't think `Parsers` needs to be exposed

Reply:
Turned out it was needed for `cabal build cardano-testnet:exe:cardano-testnet`.
In any case, I renamed `Parsers` to `Testnet.Parsers` and kept it in the exported API.